Cargo.toml

```diff
@@ -160,6 +159,8 @@ wildcard-imports = "warn"
 zero_sized_map_values = "warn"
 
 [workspace.dependencies]
+edr_test_utils = { path = "crates/edr_test_utils" }
```

Comment:
Shouldn't this always be a locally specified dev dependency?

To the best of my knowledge, [workspace dependencies are non-optional](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table). Making this a dependency requires compilation of `edr_test_utils` for non-test configurations.

Reply:
> To the best of my knowledge, [workspace dependencies are non-optional](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table)

I think that just means that you cannot mark it as optional, e.g. `edr_test_utils = { workspace = true, optional = true }` doesn't work. But dev dependencies cannot be marked as optional either and `edr_test_utils` is always used as a dev dependency, so I don't think this is a problem.

> Making this a dependency requires compilation of `edr_test_utils` for non-test configurations.

I've tried this in the branch by doing `cd crates/edr_napi && cargo clean && pnpm build` and it didn't compile `edr_test_utils`, so I think it's ok.

> Shouldn't this always be a locally specified dev dependency?

As it seems it won't force compilation of `edr_test_utils` for non-test builds, and it saves typing out the path where it's used, I kinda like this pattern (borrowed it from Foundry).